articles/moricho.re

```diff
@@ -0,0 +1,71 @@
+= Goによるコンテナランタイム自作入門
+
+== はじめに
+こんにちは、@_moricho_@<fn>{_moricho_}です。
+〜〜〜
+
+//footnote[_moricho_][@<href>{https://twitter.com/_moricho_}]
+
+== コンテナランタイムとは？
```

Comment:
**[textlint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Disallow to use "？". (ja-technical-writing/no-exclamation-question-mark)